First draft of results and discussion.

mtayler opened the pull request:
Missing anything important? I'm going to try and give it a second run down tomorrow and add anything I see should be there.

mtayler committed:
First draft of results and discussion.

trstephen commented on lab04/sections/results.tex:
```diff
@@ -1,3 +1,32 @@
 \section{Results}\label{sec:results}
-Describe the apparatus and measurement technique(s).
-Present the data.
+Four circuits were assembled and analyzed using a function generator to apply a stimulus, and an oscilloscope to measure the output of the system over the input. A detailed diagram of the circuits which produced these outputs can be referenced in the lab manual \cite{lab-manual}.
```
"to apply a stimulus" should be specific, i.e. 1kHz, 1Vpp

trstephen replied:
"measure the output of the system over the input"

Not really correct. It just displays the output and input, it doesn't do any analysis of the signals.

trstephen replied:
Schematics are included in `graphics/schematic-XXX`. You'll need to include them with the resistance values to make claims about what the expected gain should be.

trstephen commented on lab04/sections/results.tex:
```diff
@@ -1,3 +1,32 @@
+	\centering
+	\includegraphics{response-inverter}
+	\label{fig:inverter}
+	\caption{Input versus output of inverter circuit}
```
Specify which trace is the output/input. Top? Bottom?

trstephen commented on lab04/sections/discussion.tex:
```diff
@@ -1,2 +1,10 @@
 \section{Discussion}\label{sec:discussion}
-Analysis and interpretation of data.
+We confirmed frequency analysis through the Laplace transform is a valid and efficient way to analyze the response of a circuit to inputs of particular or varying frequencies. 
```
There's not really any frequency analysis occurring. You could think of negative gain as a 180deg phase shift but that's not how we present the analysis.

The laplace domain circuit is analyzed with impedances and algebraic operations rather than integro-differential operations.

trstephen commented on lab04/sections/discussion.tex:
```diff
@@ -1,2 +1,10 @@
-Analysis and interpretation of data.
+We confirmed frequency analysis through the Laplace transform is a valid and efficient way to analyze the response of a circuit to inputs of particular or varying frequencies. 
+
+The output of the inverter circuit in Fig.~\ref{fig:inverter} is of equal amplitude to the input waveform, but has a gain of approximately -1. This is the expected behaviour of an inverter circuit.
```
"... the input waveform, but has..."
Comma splice.

trstephen replied:
What is the expected gain? Refer to the schematics and formula from the intro.

trstephen commented on lab04/sections/discussion.tex:
```diff
@@ -1,2 +1,10 @@
+
+The output of the inverter circuit in Fig.~\ref{fig:inverter} is of equal amplitude to the input waveform, but has a gain of approximately -1. This is the expected behaviour of an inverter circuit.
+
+The adder circuit has an output, visible in Fig.~\ref{fig:inverter} of the same phase as the input signal, however the output signal is the result of adding the input signal to itself, having a gain of 2. This deviates from the experimental procedure of add a DC offset to the input sinusoid, however adding the input to itself confirms the adder circuit functions correctly when both inputs are tim ...
```
First sentence is run-on.

mtayler commented:
Fuck I've been trying to work on this all afternoon but my laptop keeps crashing every 5-10 minutes. Kind of 2 steps forward 1 step back.

mtayler committed:
2nd draft of results and discussion

- Added circuit schematic
- Fixed grammatical/structrual issues

trstephen commented:
FYI, Gonna hop on here and do some changes.

trstephen committed:
Third draft of discussion

- Combine schematic and response into side-by-side subfigures
- Remove equation numbers from discussion
